pkgs/by-name/re/redocly/package.nix

```diff
@@ -32,24 +34,25 @@ buildNpmPackage rec {
 
     mkdir $out/bin
     makeWrapper $out/lib/node_modules/@redocly/cli/node_modules/@redocly/cli/bin/cli.js \
-      $out/bin/redocly-cli \
+      $out/bin/redocly \
       --set-default REDOCLY_TELEMETRY off \
       --set-default CI true # Silence update messages
-  '';
 
-  installCheckPhase = ''
-    runHook preInstallCheck
-    $out/bin/redocly-cli --version
-    runHook postInstallCheck
+    # Symlink for backwards compatibility. Remove after deprecation.
```

Comment:
```suggestion
    # Symlink for backwards compatibility. Remove after NixOS 23.11
```

Reply:
This is going into unstable/24.05. I'll remove it in 24.11.